src/features/permissions/constants/permissionScopes.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * All permission scopes.
+ * The scopes should be aligned with backend
+ */
+export enum PermissionEnum {
```

Comment:
Yeah we have had this discussion a few times as well, and we ended on postfixing with `Enum` because it will give two benefits:
1. When reading the code, you immediately know that it is an Enum
2. It will not clash with any other names, we could have a component or type with the same name, so for the same reason we think types should be postfixed with `Def`.